### PR TITLE
LPS-186363 | master

### PR DESCRIPTION
**Pull request by melodywu**
https://issues.liferay.com/browse/LPS-146160

### CODE REVIEW

**Review by gaborlovasdotcom — APPROVED** [review]

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/staging/usecase/Staging.testcase [hunk 1]
```diff
@@ -268,6 +268,40 @@ definition {
 		}
 	}
 
+	@description = "This is a test for LPS-146160 : Verify Admin can enable local staging on the inherited site even site template propagation is disabled.The testcase is blocked by"
```
Comment (gaborlovasdotcom) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/staging/usecase/Staging.testcase:271]: Missing blocked ticket reference from the description

Comment (melodywu) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/staging/usecase/Staging.testcase:271]: thanks and done

**Comment by melodywu** [comment]
ci:test:sf

**Comment by gaborlovasdotcom** [comment]
ci:forward

**Pull request closed without merging**